src/runty8-core/src/input.rs

```diff
@@ -69,8 +69,8 @@ impl Input {
 
 fn key_to_button(key: Key) -> Option<Button> {
     match key {
-        Key::X => Some(Button::Cross),
-        Key::C => Some(Button::Circle),
+        Key::X | Key::Escape => Some(Button::Cross),
```

Comment:
Hmm `Escape` is currently used for switching between the editor and the game 🤔

Reply:
Per default steamdeck button is mapped to escape - It does not cause any big issue. 
As I have tested it before